src/Components/Components/src/EventCallbackFactoryBinderExtensions.cs

```diff
@@ -53,7 +67,7 @@ private static bool ConvertToIntCore(object obj, out int value)
                 return false;
             }
 
-            if (!int.TryParse(text, out var converted))
+            if (!int.TryParse(text, NumberStyles.Any, CultureInfo.CurrentCulture, out var converted))
```

Comment:
I think it's a nice idea to update these in general to use the longest overload so that it's self-documenting. We *do* care about these details.

Reply:
It's worth noting this also results in a subtle behavior change, as the `int.TryParse` overload that doesn't take an enum uses `NumberStyles.Integer` and not `NumberStyles.Any`. Concretely, this means that things like parenthesis, currency symbols or exponents will now be considered valid.

The same is true for the other types, but they often use different `NumberStyles` flags (e.g `decimal` uses `NumberStyles.Number`).

Reply:
Ah that's a really good point. I was trying to find that info and I found it wrongly it looks like. I don't intend to change the behavior of this without more discussion so I'll update it. 👍